ckanext/dcat/harvesters/rdf.py

```diff
@@ -66,27 +66,6 @@ def _get_guid(self, dataset_dict, source_url=None):
 
         return guid
 
-    def _get_existing_dataset(self, guid):
```

Comment:
I see that we moved this method to base.py, do we need to update the function call in rdf.py at line 285

Reply:
If you look on line 25 you can see that base is the parent of DCATRDFHarvester. I think we can leave the function call alone.